plugins/woocommerce/src/Blocks/BlockTypes/Reviews/ProductReviewsTitle.php

```diff
@@ -0,0 +1,98 @@
+<?php
+namespace Automattic\WooCommerce\Blocks\BlockTypes\Reviews;
+
+use Automattic\WooCommerce\Blocks\BlockTypes\AbstractBlock;
+
+/**
+ * ProductReviewsTitle class.
+ */
+class ProductReviewsTitle extends AbstractBlock {
+	/**
+	 * Block name.
+	 *
+	 * @var string
+	 */
+	protected $block_name = 'product-reviews-title';
+
+	/**
+	 * Render the block.
+	 *
+	 * @param array $attributes Block attributes.
+	 * @return string Rendered block content.
+	 */
+	protected function render( $attributes, $content, $block ) {
```

Comment:
This is based off of the comments-title render function, just making use of the actual `$product` entity.